libevm/ExtVMFace.h

```diff
@@ -90,7 +90,7 @@ struct SubState
 {
     std::set<Address> suicides;  ///< Any accounts that have suicided.
     LogEntries logs;             ///< Any logs.
-    u256 refunds;                ///< Refund counter for storage changes.
+    int64_t refunds = 0;         ///< Refund counter for storage changes.
```

Comment:
Wouldn't it be safer to use signed bigint?

Reply:
Yes, but if 63 bits are not enough I'd like to constructs a test for this case.

Reply:
Yeah. For this counter, geth uses `u64`, and parity uses `i128` (which is basically signed `u64`). If we make every opcode give maximum amount of refund as we currently have, it would still take decades of execution for that to ever overflow.

Reply:
Thanks for clarification @sorpaas.